crates/models/src/source_capture.rs

```diff
@@ -0,0 +1,64 @@
+use super::Capture;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize,  Clone, PartialEq, JsonSchema)]
+#[serde(deny_unknown_fields, rename_all = "camelCase")]
+pub enum SourceCaptureSchemaMode {
```

Comment:
Can you explain your thinking on why this is an enum vs a boolean? Are there other values that you think might make sense in the future?

My knee jerk is just that the name `collectionSchema` is a bit confusing, since flow collections don't technically have a schema. But wondering if maybe bikeshedding the names could be irrelevant, if we're able to change it to something like `set_schema_name: bool`?

Reply:
I can get on board with `collectionSchema` being ambiguous / confusing... but I don't think `set_schema_name` is any better, they both do not really convey what is going to happen through their name and they both would need documenting to make it clear.

I went with an enum to allow for future extensions to this system. We already are going to have fixed literals as schema for collections when added through the UI. We may want to have the same thing for the `sourceCapture`.

In either case I think we will need a more meaningful name